### ISSUE TITLE
Any clue to detect hand pose?

### ISSUE THREAD

**Issue by peterchen2001** [yukihiko/ThreeDPoseUnitySample#8]
Great work! This sample works fine for detecting full body, however it doesn't work well for hand pose. Any clue to detect hand pose?

**Comment by yukihiko** [comment]
Yes, the current input image is too small to know the hand shape.
Therefore, it is necessary to use other hand posture estimation, but I don't know which one is good.
